Vic2ToHoI4/Source/V2World/Country.cpp

```diff
@@ -22,71 +22,70 @@ Vic2::Country::Country(const std::string& theTag,
 		commonItems::singleDouble warExhaustionDouble(theStream);
 		warExhaustion = warExhaustionDouble.getDouble();
 	});
-	registerKeyword(std::regex("badboy"), [this](const std::string& unused, std::istream& theStream) {
+	registerKeyword("badboy", [this](const std::string& unused, std::istream& theStream) {
 		commonItems::singleDouble badboyDouble(theStream);
 		badboy = badboyDouble.getDouble();
 	});
-	registerKeyword(std::regex("prestige"), [this](const std::string& unused, std::istream& theStream) {
+	registerKeyword("prestige", [this](const std::string& unused, std::istream& theStream) {
 		commonItems::singleDouble prestigeDouble(theStream);
 		prestige = prestigeDouble.getDouble();
 	});
-	registerKeyword(std::regex("government"), [this](const std::string& unused, std::istream& theStream) {
+	registerKeyword("government", [this](const std::string& unused, std::istream& theStream) {
 		commonItems::singleString governmentString(theStream);
 		government = governmentString.getString();
 	});
-	registerKeyword(std::regex("last_election"), [this](const std::string& unused, std::istream& theStream) {
+	registerKeyword("last_election", [this](const std::string& unused, std::istream& theStream) {
 		commonItems::singleString lastElectionString(theStream);
 		lastElection = date(lastElectionString.getString());
 	});
-	registerKeyword(std::regex("domain_region"), [this](const std::string& unused, std::istream& theStream) {
+	registerKeyword("domain_region", [this](const std::string& unused, std::istream& theStream) {
 		commonItems::singleString regionString(theStream);
 		domainName = regionString.getString();
 		domainAdjective = domainName;
 	});
-	registerKeyword(std::regex("human"), [this](const std::string& unused, std::istream& theStream) {
+	registerKeyword("human", [this](const std::string& unused, std::istream& theStream) {
 		commonItems::ignoreItem(unused, theStream);
 		human = true;
 	});
-	registerKeyword(std::regex("primary_culture"),
-		 [this, &theCultureGroups](const std::string& unused, std::istream& theStream) {
-			 commonItems::singleString cultureString(theStream);
-			 primaryCulture = cultureString.getString();
-			 if (primaryCulture.substr(0, 1) == "\"")
-			 {
-				 primaryCulture = primaryCulture.substr(1, primaryCulture.size() - 2);
-			 }
-			 acceptedCultures.insert(primaryCulture);
-
-			 auto cultureGroupOption = theCultureGroups.getGroup(primaryCulture);
-			 if (cultureGroupOption)
-			 {
-				 primaryCultureGroup = *cultureGroupOption;
-			 }
-			 else
-			 {
-				 primaryCultureGroup.clear();
-			 }
-		 });
-	registerKeyword(std::regex("culture"), [this](const std::string& unused, std::istream& theStream) {
+	registerKeyword("primary_culture", [this, &theCultureGroups](const std::string& unused, std::istream& theStream) {
+		commonItems::singleString cultureString(theStream);
+		primaryCulture = cultureString.getString();
+		if (primaryCulture.substr(0, 1) == "\"")
```

Comment:
' for char

Reply:
No can do. std::string doesn't have a comparison operator that takes a character. It needs a string.

Reply:
Oh what I meant was it's probably faster just to check the first character [0] or to find(char) == 0 than to invoke substr for that. I know codacy game me trouble about the same thing.